spec/acceptance/spiped_spec.rb

```diff
@@ -41,10 +41,15 @@
 
   describe 'redis over tunnel' do
     if fact('hostname') == 'spipedserver'
+      redis_service = if fact('operatingsystemmajrelease') == '8'
```

Comment:
This will probably break on EL8 once it's out

Reply:
ah yes.  It's only in the acceptance test code, but will need adjusting when eventually centos 8 is out and we want to test on that too.